Library/Formula/openwsman.rb

```diff
@@ -18,6 +18,8 @@ class Openwsman < Formula
   depends_on "pkg-config" => :build
   depends_on "openssl"
 
+  patch :DATA
```

Comment:
I will, but since it will involve fixing up their CMake scripts, it will take a _lot_ more time than this little patch took. It may be just as well, since this patch isn't working anyway.